src/asf_tools/flood_map.py

```diff
@@ -222,30 +224,102 @@ def make_flood_map(out_raster: Union[str, Path],  vv_raster: Union[str, Path],
               epsg_code=epsg, dtype=gdal.GDT_Float64, nodata_value=False)
 
 
-def main():
+def _get_cli(interface: Literal['hyp3', 'main']) -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
-    parser.add_argument('out_raster',
-                        help='File flood depth map will be saved to.')
-    parser.add_argument('vv_raster',
-                        help='Sentinel-1 RTC GeoTIFF raster, in power scale, with VV polarization')
-    parser.add_argument('water_extent_map',
-                        help='Hyp3-Generated water extent raster file.')
-    parser.add_argument('hand_raster',
-                        help='Height Above Nearest Drainage (HAND) GeoTIFF aligned to the RTC rasters. '
-                             'If not specified, HAND data will be extracted from a Copernicus GLO-30 DEM based HAND.')
-
-    parser.add_argument('--estimator', type=str, default='iterative', choices=['iterative', 'logstat', 'nmad', 'numpy'],
+
+    available_estimators = ['iterative', 'logstat', 'nmad', 'numpy']
+    if interface == 'hyp3':
+        parser.add_argument('--bucket')
+        parser.add_argument('--bucket-prefix', default='')
+        parser.add_argument('--wm-raster',
+                            help='Water map GeoTIFF raster, with suffix `_WM.tif`.')
+        available_estimators += ['None']
```

Comment:
Alternatively, we could change `available_estimators` to a `tuple`.
```suggestion
        available_estimators.append('None')
```